appveyor.yml

```diff
@@ -1,16 +1,14 @@
 version: 1.1.0.{build}
 environment:
-  OPENSSL_INCLUDE_DIR: C:\OpenSSL\include
-  OPENSSL_LIB_DIR: C:\OpenSSL\lib
+  OPENSSL_INCLUDE_DIR: C:\OpenSSL-Win64\include
```

Comment:
Did appveyor change or is this us breaking ourself?

Reply:
We were referring to a package that's no longer hosted (a few lines below). AppVeyor images added 1.0.2 installs of OpenSSL: https://www.appveyor.com/docs/build-environment/#pre-installed-software.